README.md

```diff
@@ -404,6 +404,6 @@ Terms and conditions for use, reproduction and distribution are under the [Apach
 </a>
 </center>
 <br>
+<p align="right"><a href="#top">Back to top</a></p>
```

Comment:
why you duplicates this if not needed remove

Reply:
we are not looking for more and more pr more than the quality of the pr. For us quality of pr is as important as pr merging for you.